pydantic2ts/cli/script.py

```diff
@@ -223,6 +221,8 @@ def main(module: str, output: str, json2ts_cmd: str = "json2ts") -> None:
     """
     CLI entrypoint to run :func:`generate_typescript_defs`
     """
+    logging.basicConfig(level=logging.DEBUG, format="%(asctime)s %(message)s")
```

Comment:
Moving this here makes it so calling `pydantic2ts` will still print debug logs but importing `generate_typescript_defs` won't print by default

Reply:
Thank you! Doing the work for me AND teaching me something very useful/relevant, I can't ask for more than that 😛

But I do really appreciate it, and your solution makes sense --  merging this in now and then I'll release it along with a few other minor changes